infrastructure/environments/dev.tfvars

```
@@ -17,3 +17,6 @@ vpc = {
   database_subnet_b = "10.170.202.0/24"
   database_subnet_c = "10.170.203.0/24"
 }
+
+rds_serverless_min_capacity = 2
+rds_serverless_max_capacity = 4
```

Can we leave this environment as a single node within the cluster? I.e. 
```suggestion
rds_serverless_min_capacity = 1
rds_serverless_max_capacity = 1
```